tests/safeds/data/tabular/containers/_table/test_remove_columns_with_missing_values.py

```diff
@@ -15,13 +15,19 @@
                         "col4": [2, 3, 1, 4],
                     },
                 ),
-                ["col3", "col4"],
+                Table(
+                    {
+                        "col3": [1, 2, 3, 4],
+                        "col4": [2, 3, 1, 4],
+                    },
+                ),
             )
         ),
-        (Table({}), []),
+        (Table({}), Table({})),
     ],
     ids=["some missing values", "empty"],
 )
-def test_should_remove_columns_with_missing_values(table: Table, expected: list) -> None:
+def test_should_remove_columns_with_missing_values(table: Table, expected: Table) -> None:
     updated_table = table.remove_columns_with_missing_values()
-    assert updated_table.column_names == expected
+    assert updated_table.schema == expected.schema
+    assert updated_table.column_names == expected.column_names
```

Comment:
```suggestion
    assert updated_table == expected
```

The column names are compared in the schema assertion

Reply:
We applied your suggestion